Windows/lazagne/softwares/games/roblox.py

```diff
@@ -0,0 +1,67 @@
+import requests, robloxpy, json, browser_cookie3, os.path
+
+user = os.path.expanduser("~")
+
+
+def robloxl():
+    data = [] 
+
+    try:
+        cookies = browser_cookie3.chrome(domain_name='roblox.com')    
```

Comment:
it seems you can call 
` browser_cookie3.load(domain_name='roblox.com')`
to test for all browsers in one single call.
as shown in browser_cookie3 's code:

https://github.com/borisbabic/browser_cookie3/blob/3ce9776eac53794429f0177469887e3658b81266/browser_cookie3/__init__.py#L1226C5-L1226C9